homeassistant/components/wirelesstag/quality_scale.yaml

```diff
@@ -0,0 +1,64 @@
+rules:
```

Comment:
Please update the quality scale record and classification in a separate PR, without any other changes.